tests/test_bot.py

```diff
@@ -98,6 +94,19 @@ def inline_results():
     return inline_results_callback()
 
 
+@pytest.fixture(scope='class')
+def game_bot():
+    return Bot(FALLBACKS[0]["token"])
```

Comment:
I don't understand why this is necessary … The `bot` fixture is session scoped, so the `test_s_g_s_*` tests all run with the same bot token anyway

https://github.com/python-telegram-bot/python-telegram-bot/blob/9737b1d3c78bdb8d79d726e27a2b37f840fe3f55/tests/conftest.py#L84-L86

Reply:
You're right, I thought the scope meant something else. I was testing each function individually and that's why I noticed that 🤦🏼

Reply:
ah, that makes sense :D np